lib/node_modules/@stdlib/_tools/scaffold/math-iter-unary/scripts/scaffold.sh

```diff
@@ -147,7 +147,7 @@ unary_pkg_path="stdlib/math/base/special"
 unary_pkg="${unary_pkg_path}/${unary_pkg_alias}"
 
 # Define the REPL text main export description (note: 4 space indent and wrap at 80 characters):
-repl_text_main_desc=$(echo -n "${main_desc}" | "${wrap}")
+repl_text_main_desc=$(echo "${main_desc}" | "${wrap}")
```

Comment:
```suggestion
repl_text_main_desc=$(printf "%s" "${main_desc}" | "${wrap}")
```

We can replace with `printf`, I believe.